pkg/middleware/jwt_session.go

```diff
@@ -15,19 +15,21 @@ import (
 
 const jwtRegexFormat = `^ey[a-zA-Z0-9_-]*\.ey[a-zA-Z0-9_-]*\.[a-zA-Z0-9_-]+$`
 
-func NewJwtSessionLoader(sessionLoaders []middlewareapi.TokenToSessionFunc) alice.Constructor {
+func NewJwtSessionLoader(sessionLoaders []middlewareapi.TokenToSessionFunc, denyInvalidJwts bool) alice.Constructor {
```

Comment:
Nit, should the name be `denyInvalidJWTs`? I think that's how I would have camel-cased this at least

Reply:
Sure, I think I was following `jwtRegex` etc, but that makes sense as it's at the beginning. Updating